bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -9,6 +9,10 @@
 		<label>Somfy Tahoma Bridge</label>
 		<description>Somfy Tahoma bridge enabling communication with Somfy devices</description>
 
+		<channels>
+			<channel id="scenes" typeId="scenes"></channel>
+		</channels>
+
```

Comment:
I still think a better place for this channel is Gateway, not Bridge. The Bridge thing is responsible for the cloud communication and should stay as is, without any channels. If something aggregates the devices in the scenarios, it is the Gateway.

Reply:
Ok, why not. I will move the channel. It makes the thing just a little more difficult.

Reply:
Done